OrbitCore/Capture.cpp

```diff
@@ -35,21 +34,19 @@ void Capture::SetTargetProcess(std::shared_ptr<Process> process) {
   if (process != GTargetProcess) {
     GSamplingProfiler = std::make_shared<SamplingProfiler>(process);
     GTargetProcess = std::move(process);
-    GSelectedFunctionsMap.clear();
   }
 }
 
-ErrorMessageOr<void> Capture::StartCapture() {
+ErrorMessageOr<void> Capture::StartCapture(
+    uint64_t process_id, std::string process_name,
+    absl::flat_hash_map<uint64_t, FunctionInfo> selectedFunctions) {
```

Comment:
```suggestion
    absl::flat_hash_map<uint64_t, FunctionInfo> selected_functions) {
```

Reply:
Thanks, I changed it in the last commit.